diff --git a/node_modules/react-native-screen-corner-radius/android/build.gradle b/node_modules/react-native-screen-corner-radius/android/build.gradle
index 48d7647..db90387 100644
--- a/node_modules/react-native-screen-corner-radius/android/build.gradle
+++ b/node_modules/react-native-screen-corner-radius/android/build.gradle
@@ -1,17 +1,15 @@
 buildscript {
-  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
-  def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['ScreenCornerRadius_kotlinVersion']
+    ext.kotlin_version = '1.6.21'  // Update Kotlin Version ✅
 
-  repositories {
-    google()
-    jcenter()
-  }
+    repositories {
+        google()
+        mavenCentral()
+    }
 
-  dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
-    // noinspection DifferentKotlinGradleVersion
-    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-  }
+    dependencies {
+        classpath("com.android.tools.build:gradle:7.0.2") // Ensure Gradle is updated
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version") // Updated Kotlin plugin ✅
+    }
 }
 
 apply plugin: 'com.android.library'
