diff --git a/node_modules/react-native-minimizer/.bun-tag b/node_modules/react-native-minimizer/.bun-tag
new file mode 100644
index 0000000..539a0b0
--- /dev/null
+++ b/node_modules/react-native-minimizer/.bun-tag
@@ -0,0 +1 @@
+rainbow-me-react-native-minimizer-26ae6dc
\ No newline at end of file
diff --git a/node_modules/react-native-minimizer/android/build.gradle b/node_modules/react-native-minimizer/android/build.gradle
index 43e070f..d227941 100644
--- a/node_modules/react-native-minimizer/android/build.gradle
+++ b/node_modules/react-native-minimizer/android/build.gradle
@@ -20,7 +20,7 @@ def safeExtGet(prop, fallback) {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 buildscript {
     // The Android Gradle plugin is only required when opening the android folder stand-alone.
